workingjubilee.md

```diff
@@ -1,7 +1,7 @@
 Hello,
 
-My name is Jubilee! It's a pleasure to meet you.
+My name is Jubilee! It's a pleasure to EAT YOU! Muahaha!
 
-I'm an artificially enhanced human from the 21st Century.
+I'm a cannibalistic supermutant neobarbarian from the 24th Century.
 
-I like reading up on neuroscience papers. Partly, it's because I'm doing a lot of things to my neurology already that might have all sorts of interesting effects. But also it's because a lot of possibilities are opening up right now with how we can modify our brains, and I'm interested in seeing where those go.
+I like scavenging the postnuclear wasteland, gambling for precious things in bizarre contests, and holding extravagant parties after successful raids on supply depots.
```

Comment:
Well these are certainly the more creative answers i've seen for this project lol. 

Line 7 makes me wonder if you're a Fallout fan.